atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -326,12 +323,12 @@ protected void initialize(boolean asyncInitialize) {
     }
 
     private void tryInitialize() {
-        boolean supportsCas = !configManager.getConfig().scyllaDb()
+        boolean supportsCas = !config.scyllaDb()
```

Comment:
We support ScyllaDB?!?!

Reply:
Only for @clockfort's nefarious side-projects...

Reply:
😸 not officially, but once we move onto a CQL-only cassandra world it's probably a matter of time